packages/devtools_app/lib/src/http/http_request_data.dart

```diff
@@ -507,6 +68,11 @@ class DartIOHttpRequestData extends HttpRequestData {
         .then((updated) => _request = updated);
   }
 
+  static List<Cookie> _parseCookies(List<dynamic>? cookies) {
```

Comment:
Should this be `List<dynamic>`? Do we not have a static type to use here?

Reply:
changed to `List<String>`